cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -332,6 +332,17 @@ data:
       </metric>
     </match>
 
+    # Trim the entries which exceed slightly less than 100KB, to avoid
+    # dropping them. It is a necessity, because Stackdriver only supports
+    # entries that are up to 100KB in size.
+    <filter kubernetes.**>
+      @type record_transformer
+      enable_ruby true
+      <record>
+        log ${record['log'].length > 100000 ? '[The entry was trimmed] ' + record['log'][0:100000 - 25] : record['log']}
```

Comment:
There's no need to use `0:100000 - 25` because `100000` is already smaller than the 100k limit (`102400`).

Reply:
Will this message break JSON encoded log lines?

Reply:
Ack, agree that it's better to keep it clean

Reply:
> Will this message break JSON encoded log lines?
 
Yes, it will. It is a conscious decision, because the alternative is dropping this JSON entry